modules/cnf-troubleshooting-common-ptp-operator-issues.adoc

```diff
@@ -0,0 +1,126 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-ptp.adoc
+
+[id="cnf-troubleshooting-common-ptp-operator-issues_{context}"]
+= Troubleshooting common PTP Operator issues
+
+Troubleshoot common problems with the PTP Operator by performing the following steps.
+
+.Prerequisites
+
+* Install the {product-title} CLI (`oc`).
+* Log in as a user with `cluster-admin` privileges.
+* Install the PTP Operator on a bare-metal cluster with hosts that support PTP.
+
+.Procedure
+
+. Check the operator and operands are successfully deployed in the cluster.
++
+[source,terminal]
+----
+$ oc get pods -n openshift-ptp
+----
++
+.Example output
+[source,terminal]
+----
+NAME                            READY   STATUS    RESTARTS   AGE
+linuxptp-daemon-5j265           3/3     Running   0          24h
+linuxptp-daemon-s72q2           3/3     Running   0          24h
+ptp-operator-6b8dcbf7f4-czff8   1/1     Running   0          10d
+----
++
+[NOTE]
+====
+When the event bus is enabled, the number of ready linuxptp-daemon pods is `3/3`. If the event bus is not enabled, `2/2` is displayed.
+====
+
+. Check that supported hardware is found in the cluster.
++
+[source,terminal]
+----
+$ oc -n openshift-ptp get nodeptpdevices.ptp.openshift.io
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                  AGE
+control-plane-0.example.com           10d
+control-plane-1.example.com           10d
+compute-0.example.com                 10d
+compute-1.example.com                 10d
+compute-2.example.com                 10d
+----
+
+. Check the available PTP network interfaces:
++
+[source,terminal]
+----
+$ oc -n openshift-ptp get nodeptpdevices.ptp.openshift.io <node_name> -o yaml
+----
++
+where:
++
+<node_name>:: is the node you want to query, for example, `compute-0.example.com`.
```

Comment:
Does it matter which one? I'm not sure which one to pick. Will the user know?

Reply:
I've updated the procedure to clarify that you select the node to query by running `oc get pods -n openshift-ptp -o wide` to show the corresponding node for the `linuxptp-daemon`.